test/vagrant/functional/test_func.py

```diff
@@ -152,3 +152,8 @@ def test_multi_node(temp_dir):
         content = f.read()
         assert "instance-1" in content
         assert "instance-2" in content
+
+    # Validate Vagrantfile
+    vagrantdir = os.path.dirname(vagrantfile)
+    result = vagrant.Vagrant().validate(vagrantdir)
+    assert result.returncode == 0
```

Comment:
hm. Why is it needed ? iirc, the plugin code should already be running validate

Reply:
'Cause I'm a noob and didn't realize `validate` was being called.
I've removed it.